crates/bevy_ecs/src/system/query/mod.rs

```diff
@@ -181,12 +181,21 @@ impl<'a, Q: WorldQuery, F: QueryFilter> Query<'a, Q, F> {
             .map_err(QueryError::ComponentError)
     }
 
-    /// Returns an array containing the entities in this query that had the given component
+    /// Returns an array containing the `Entity`s in this `Query` that had the given `Component`
```

Comment:
You could use doc links here if you wanted.

Reply:
I considered this, but there were a few reasons for not doing it:
- Quickly glancing at other doc comments I didn't see doc links being used, so this function might be an outlier if it used doc links.
- Links to `Entity` and `Component` are already available directly above the documentation in the function definition and you're already on the doc page for `Query`.

Let me know if I should add them anyway. :)